angular-bind-html-compile.js

```diff
@@ -6,15 +6,17 @@
     module.directive('bindHtmlCompile', ['$compile', function ($compile) {
         return {
             restrict: 'A',
-            link: function (scope, element, attrs) {
```

Comment:
Can we retain the spacing after the `function` keyword? That will keep the code style consistent with the rest of the file and introducing less changes.

We currently don't have any code style rules for this repository, but I've made an issue to add them in #4